nni/nas/execution/common/utils.py

```diff
@@ -32,3 +34,35 @@ def mutation_dict_to_summary(mutation: dict) -> dict:
 def get_mutation_summary(model: Model) -> dict:
     mutation = get_mutation_dict(model)
     return mutation_dict_to_summary(mutation)
+
+def init_execution_engine(config, port, url_prefix) -> AbstractExecutionEngine:
```

Comment:
Shall we put this in `nni/nas/execution/api.py`?

Reply:
the reason is?